build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -367,6 +367,61 @@ function ensurePrivatePackagesComputed(): void {
 	});
 }
 
+/**
+ * Parses command line string into arguments following OS quote rules
+ *
+ * @param commandLine - complete command line as a simple string
+ * @param onlyDoubleQuotes - only consider double quotes for grouping
+ * @returns array of ordered pairs of resolved `arg` strings (quotes and escapes resolved)
+ *  and `original` corresponding strings.
+ */
+function parseArgs(commandLine: string, { onlyDoubleQuotes }: { onlyDoubleQuotes: boolean }) {
```

Comment:
I didn't realize you could destructure function arguments like this. Cool!

Reply:
Oh yes. Very good for boolean options to force naming them.